src/public/assets/images/global/search.svg

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
I believe that both search and arrow icons are implemented as CSS in the ScandiPWA core.